paddle/fluid/inference/tensorrt/engine.h

```diff
@@ -538,6 +540,17 @@ class TensorRTEngine {
     }
   }
 
+  void getEngineInfo() {
+#if IS_TRT_VERSION_GE(8200)
+    auto infer_inspector_ = infer_engine_->createEngineInspector();
+    infer_inspector_->setExecutionContext(context());
+    VLOG(3) << infer_inspector_->getEngineInformation(
+        nvinfer1::LayerInformationFormat::kJSON);
```

Comment:
A trailing underscore is usually used for class/struct member. Here we should not use it.

Reply:
renamed variable infer_inspector_ to infer_inspector